XSerializer/CustomSerializer.cs

```diff
@@ -111,7 +111,7 @@ public CustomSerializer(EncryptAttribute encryptAttribute, IXmlSerializerOptions
 
                         if (serializableProperties.Count(p => p.IsListDecoratedWithXmlElement) > 1)
                         {
-                            throw new InvalidOperationException("More than one list property is decorated with [XmlElement] attribute.");
+                           // throw new InvalidOperationException("More than one list property is decorated with [XmlElement] attribute.");
```

Comment:
are you sure this is enough?
cause it will just not throw, but don't we need any other code changes?